vm/src/stdlib/json.rs

```diff
@@ -188,31 +188,31 @@ impl<'de> Visitor<'de> for PyObjectDeserializer<'de> {
     }
 }
 
+pub fn ser_pyobject(vm: &mut VirtualMachine, obj: &PyObjectRef) -> serde_json::Result<String> {
```

Comment:
Please do not expose functions like this. I would prefer to figure out what went wrong, and fix that instead.

Reply:
What about as a method on VM? `json` could be a `pub(crate) mod` and it wouldn't really be exposed.

Reply:
I don't really agree with @windelbouwman here - this isn't an implementation detail, it's the actual python function so it's inevitable to expose these at some point to allow developing native extensions without unnecessary overhead.

> The code would also be faster because it's not looking up a function and loading it from a module, it's just calling a Rust function.

Yep! :+1: